lms/djangoapps/grades/models.py

```diff
@@ -176,6 +180,7 @@ def bulk_create(cls, block_record_lists):
                 blocks_json=brl.json_value,
                 hashed=brl.hash_value,
                 course_id=brl.course_key,
+                version=VISIBLE_BLOCKS_VERSION,
             )
             for brl in block_record_lists
         ])
```

Comment:
What about `bulk_read`?  For forward compatibility, we'll want older versions of the code to only retrieve data with the version that it knows how to handle, right?  Given this, it probably makes sense to always query with version, and always use the latest version to look up entries, yes?